exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,7 @@
-const name = " Daniel  ";
+const name = "          Daniel             ";
+var nameLength = name.length;
+let message = "Hello, my name is ";
+
+console.log(message.trim() + name.trim() + " and my name is " + nameLength + " characters long");
```

Comment:
as a result of doing `message.trim()` what is now missing between `message` and `name` in your `console.log`?